src/components/orderItem.js

```diff
@@ -61,12 +73,16 @@ class OrderItem extends React.Component {
             
          </div>
       </div>
-              
-                
-                
+                     
             </div>
         )
     }
 }
 
-export default connect()(OrderItem);
+
+const stateMappings = state => ({})
+const actionMappings = {
+    returnExchangeDetails
+}
+
+export default connect(stateMappings, actionMappings)(OrderItem);
```

Comment:
Redux should not be necessary for a component at a lower level (drop down, line item).

Anything that appears multiple times within the website *probably* shouldn't use redux directly. Redux is for application-level state -- that is to say, data that should be widely available in the application.

Changing the details of one line item should trigger a prop to run a function in a higher order component (e.g. order details). High order components can then link to redux to set application-level state, based on changes from lower order components like OrderItem, Dropdown, and Options.

All of this should be able to be swapped out for:
```suggestion
export default OrderItem;
```